pkgs/development/python-modules/karton-core/default.nix

```diff
@@ -8,16 +9,20 @@
 
 buildPythonPackage rec {
   pname = "karton-core";
-  version = "4.4.1";
+  version = "5.0.0";
 
   src = fetchFromGitHub {
     owner = "CERT-Polska";
     repo = "karton";
     rev = "refs/tags/v${version}";
-    sha256 = "sha256-smgKrFexuL0bgt/1Ikm1tpSGPJNJm7Ko68iZn3AQw5E=";
+    hash = "sha256-0B2u2xnrGc3iQ8B9iAQ3fcovQQCPqdFsn5evgdDwg5M=";
   };
 
-  propagatedBuildInputs = [ minio redis ];
+  propagatedBuildInputs = [
+    boto3
+    minio
```

Comment:
Is there any reason to keep `minio` in dependencies? It was removed and replaced by `boto3`

Reply:
Was just looking for new stuff and missed to remove it.